test/mako/alerter/alerter.go

```diff
@@ -61,7 +61,7 @@ func (alerter *Alerter) HandleBenchmarkResult(
 			summary := fmt.Sprintf("%s\n\nSee run chart at: %s\n\nSee aggregate chart at: %s",
 				output.GetSummaryOutput(),
 				output.GetRunChartLink(),
-				fmt.Sprintf("mako.dev/benchmark?benchmark_key=%s&tseconds=604800", benchmarkKey),
+				fmt.Sprintf("https://mako.dev/benchmark?benchmark_key=%s&tseconds=604800", benchmarkKey),
```

Comment:
```suggestion
				"https://mako.dev/benchmark?tseconds=604800&benchmark_key="+benchmarkKey,
```
If you want to keep sprintf, I'd highly recommend 60480 be defined as a constant `7*24*60*60`.

Reply:
Done.